packages/quantic/cypress/integration/sort/sort.cypress.ts

```diff
@@ -0,0 +1,111 @@
+import {configure} from '../../page-objects/configurator';
+import {
+  InterceptAliases,
+  interceptSearch,
+  interceptSearchIndefinitely,
+} from '../../page-objects/search';
+import {SortExpectations as Expect} from './sort-expectations';
+import {SortActions as Actions} from './sort-actions';
+import {SearchExpectations} from '../search-expectations';
+
+describe('quantic-sort', () => {
+  const sortUrl = 's/quantic-sort';
+
+  const sortOptions = ['relevancy', 'date descending', 'date ascending'];
+  const sortOptionLabels = ['Relevancy', 'Newest', 'Oldest'];
+
+  function visitSort(waitForSearch = true) {
+    interceptSearch();
+    cy.visit(sortUrl);
+    configure();
+    if (waitForSearch) {
+      cy.wait(InterceptAliases.Search);
+    }
+  }
+
+  function loadFromUrlHash(urlHash: string) {
+    interceptSearch();
+    cy.visit(`${sortUrl}#${urlHash}`);
+    configure();
+  }
+
+  function setupWithPauseBeforeSearch() {
+    interceptSearchIndefinitely();
+    cy.visit(sortUrl);
+    configure();
+  }
+
+  it('should not render before results have returned', () => {
+    setupWithPauseBeforeSearch();
+
+    Expect.displaySortDropdown(false);
+  });
+
+  describe('when loading default sort', () => {
+    it('should work as expected', () => {
+      visitSort();
+
+      Expect.displaySortDropdown(true);
+      Expect.displayLocalizedLabel('Sort by');
+
+      Actions.openDropdown();
+
+      Expect.containsOptions(sortOptions);
+      Expect.displayLocalizedOptionLabels(sortOptions, sortOptionLabels);
+      Expect.selectedOption(sortOptions[0], true);
+    });
+  });
+
+  describe('when selecting a sort option', () => {
+    sortOptions.slice(1).forEach((option) => {
+      it(`should update the shown selected option to ${option}`, () => {
```

Comment:
The 3 expectations could be validated using a single test :smiley: 

    it(`sorts value using ${option}`, () => {
      visitSort();

      Actions.selectOption(option);

      Expect.selectedOption(option, true);
      SearchExpectations.sortedBy(option); // I do like to use this form: Expect.search.sortedBy(option);
      Expect.logSortResults(option);
    });

Reply:
Yeah I think I'm just still thinking the "old" way where I'm concerned about the granularity of the report.